GetIntoTeachingApi/Models/TeachingEvent.cs

```diff
@@ -70,6 +70,11 @@ public enum EventType
         public Guid? BuildingId { get; set; }
         public bool IsVirtual => IsOnline && !string.IsNullOrWhiteSpace(Building?.AddressPostcode);
 
+        // The department refers to 'virtual' events as "in-person" (as
+        // well as offline events), so whilst virtual events are in fact online,
+        // they are deemed in-person here for consistency.
+        public bool IsInPerson => !IsOnline || IsVirtual;

```

Comment:
Useful comment here, couldn't have worked out what was going on without it 😓

Reply:
Yeah it's an odd one, I was going to go with `IsIntendedToBeInPerson` which fits the virtual events but not so much the offline events. I figure keeping it consistent with the language in use just now is probably the safest bet.